app/src/androidTest/java/org/systers/mentorship/view/fragments/LogoutTest.kt

```diff
@@ -0,0 +1,64 @@
+package org.systers.mentorship.view.fragments
+
+import android.content.ComponentName
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.intent.rule.IntentsTestRule
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.ActivityTestRule
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.systers.mentorship.R
+import org.systers.mentorship.view.activities.LoginActivity
+import org.systers.mentorship.view.activities.SettingsActivity
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class LogoutTest {
+
+    // Start the SettingsActivity
+    @get:Rule
+    val activityTestRule = ActivityTestRule(SettingsActivity::class.java)
+
+    // Rule to define the Intents
+    @Rule
+    @JvmField
+    var intentsRule = IntentsTestRule(LoginActivity::class.java)
+
+    /*
+    * Test to check if the Logout Dialog is opening on click
+    * */
+    @Test
+    fun check_of_logout_dialog_is_opening() {
```

Comment:
```suggestion
    fun check_if_logout_dialog_is_opening() {
```
Please fix the typo. Rest of the PR looks good 🙌

Reply:
Done